packages/preview/illc-mol-thesis/0.1.0/src/main.typ

```diff
@@ -0,0 +1,76 @@
+#let titleSize = 21pt
+#let subtitleSize = 14pt
```

Comment:
Public variables generally use kebab-case names in Typst, so I would suggest to either mark these as private (by moving them in the scope of the function, prefixing them with a `_`, or making another entrypoint that only re-exports the `titlepage` function from this file), or renaming them.

```suggestion
#let title-size = 21pt
#let subtitle-size = 14pt
```

(and change it in the rest of the code as well).

Reply:
Thanks! Addressed in d1abee7. I apply both tips, actually. Where can I learn about Typst's naming conventions? I could not find anything in your docs.

Reply:
This is not really documented I think, it is just implicit because that's what the standard library and a vast majority of package use. But I agree that it could be written down somewhere.